ghost/tinybird-dedicated/scripts/exec_test.sh

```
@@ -0,0 +1,58 @@
+        rm $tmpfile
+        return 1
+    fi
+    echo ""
+}
+export -f run_test
+
+fail=0
+find ./tests -name "*.test" -print0 | xargs -0 -I {} -P 4 bash -c 'run_test "$@"' _ {} || fail=1
+
+if [ $fail == 1 ]; then
+  exit -1;
```

Shellcheck (0.10.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)